public/stylesheets/style.css

```diff
@@ -139,3 +133,34 @@ ul {
     font-size: 30px;
     padding-bottom: 20px;
 }
+
+/* about page */
+.about-body {
+	background: url(../images/oregon-stadium.jpg);	 	 
+	background-position: center center;
+	background-attachment: fixed;
+	-webkit-background-size: cover;
+	-moz-background-size: cover;	
+	-o-background-size: cover;
+	background-size: cover;
+	background-color: #211922;
+	margin: 0;
+	padding: 0;
+}
+.about-center {
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 120px;
+	max-width: 600px;
+	text-align: center;
+	background-color: rgba(0, 0, 0, 0.6);
+	color: white;
+	border-radius: 5px;
```

Comment:
i dont know what this will say in the end, but I prefer border-radius:122px for more circular look